proto/wayfire-information.xml

```diff
@@ -53,7 +53,7 @@
       <description summary="Export information about a view to a client">
 	Provide client with information about a view.
       </description>
-      <arg name="view_id" type="int" summary="view wayfire ID"/>
+      <arg name="view_id" type="uint" summary="view wayfire ID"/>
```

Comment:
This change should be in a separate patch.

Reply:
Done!